src/renderer/components/cluster-icon/cluster-icon.tsx

```diff
@@ -48,7 +71,7 @@ export class ClusterIcon extends React.Component<Props> {
         )}
         {icon && <img src={icon} alt={name}/>}
         {!icon && <Hashicon value={clusterId} options={options}/>}
-        {showErrors && isAdmin && eventCount > 0 && (
+        {showErrors && eventCount > 0 && !isActive && (
```

Comment:
So non-admins now are allowed to see events?

Reply:
Yes, idk why we checked `isAdmin` here.. previously `eventCount` was only done for admins.

Reply:
Okay sounds good.